### PR TITLE
[T2A5][T13-A2]William Hawken

### PR DESCRIPTION
**Pull request by Torasian**

### CODE REVIEW

**Comment by damithc** [comment]
@Torasian pls update PR name with your team ID

**Comment by Torasian** [comment]
Hi sir,

We haven't resided out team number in our tutorial yet, so should I update when I find out?

Kind regards,

William Hawken

On 24 Aug 2016 9:42 p.m., "Damith C. Rajapakse" notifications@github.com wrote:

@Torasianhttps://github.com/Torasian pls update PR name with your team ID

**Comment by okkhoy** [comment]
@Torasian Your PR id has too many spaces :-P
[T2A5][T13-A2]William Hawken would be the right one!

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1077,15 +1100,26 @@ private static boolean isPersonEmailValid(String email) {
      */
     private static String getUsageInfoForAllCommands() {
         return getUsageInfoForAddCommand() + LS
+        		+ getUsageInfoForSortCommand() + LS
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1153]: errr! indentation is off here

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1077,15 +1100,26 @@ private static boolean isPersonEmailValid(String email) {
...
                 + getUsageInfoForFindCommand() + LS
                 + getUsageInfoForViewCommand() + LS
                 + getUsageInfoForDeleteCommand() + LS
                 + getUsageInfoForClearCommand() + LS
                 + getUsageInfoForExitCommand() + LS
                 + getUsageInfoForHelpCommand();
     }
-
     /**
+     * Builds string for showing 'sort' command usage instruction
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1162]: good to see you have updated the header comments and help on this one.

please take note, you have added a new functionality here, hence you need to update the tests as well!!

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -368,20 +381,55 @@ public static String executeCommand(String userInputString) {
...
+    	
+    	
     	Collections.sort(allPersons, new Comparator<HashMap<PersonProperty, String>>(){ 
             public int compare(HashMap<PersonProperty, String> one, HashMap<PersonProperty, String> two) { 
                 return one.get("get_name").compareTo(two.get("get_name"));
             } 
     });
-		// TODO Auto-generated method stub
-		return null;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:430]: unnecessary blank spaces here

File: src/seedu/addressbook/AddressBook.java [hunk 4]
```diff
@@ -1100,6 +1148,7 @@ private static boolean isPersonEmailValid(String email) {
      */
     private static String getUsageInfoForAllCommands() {
         return getUsageInfoForAddCommand() + LS
+        		+ getUsageInForEditCommand() + LS
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1152]: please check the indentation

**Comment by okkhoy** [comment]
@Torasian 
- in general good work on this. keep it up!
- ensure next time you add a new function, you need to add tests as well!

**Comment by okkhoy** [comment]
btw, please use your full name as it appears in IVLE in the PR titles

**Pull request closed without merging**